src/elements/content-sidebar/SidebarNavButton.js

```diff
@@ -22,12 +22,12 @@ const SidebarNavButton = ({ children, interactionTarget, onNavigate, sidebarView
     const sidebarPath = `/${sidebarView}`;
 
     return (
-        <Tooltip position="middle-left" text={tooltip}>
-            <Route path={sidebarPath}>
-                {({ match }) => {
-                    const isToggle = !!match && match.isExact;
+        <Route path={sidebarPath}>
+            {({ match }) => {
+                const isToggle = !!match && match.isExact;
 
-                    return (
+                return (
+                    <Tooltip position="middle-left" text={tooltip}>
                         <NavButton
```

Comment:
NavButton also has <Route> in it, should tooltip be inside there around the actual component?

Reply:
It worked as-is when I tested it locally. The NavButton component forwards all its props to its child, which may explain why it works properly.